javascript/newer-JS-features.md

```diff
@@ -0,0 +1,219 @@
+# Modern Features of JavaScript
+
+### Projected Time
+
+About 2 hours
+
+- 45 minutes for the video walkthrough of the slides
+- 20 minutes for the Group Practice
+- 10 minutes for Check for Understanding
+
+### Prerequisites
+
+- Techtonica's JS workshop
+- [Basic JS practice](https://github.com/Techtonica/curriculum/blob/main/javascript/basic-js-practice.md)
+
+
+### Motivation
+
+ECMAScript 2015, or ES6 introduced many important and useful features that modernized the JavaScript language and helps us to speed the development process. 
+
+### Objectives
+
+**Participants will be able to:**
+
+- Take advantage of ES6's new defalut params.
+- Use spread operator with arrays, functions and objects.
+- Know the difference between spread and rest operator. 
+- Use of Destructured Assignment
+
+### Supplemental Materials
+
+- [MDN has tons of info](https://developer.mozilla.org/bm/docs/Web/JavaScript)
+- [The Modern JavaScript Tutorial](https://javascript.info/)
+- [ES6 Tutorial](https://www.youtube.com/watch?v=WZQc7RUAg18)
+
+
+### Lesson
+
+[Modern JS Features slides](https://docs.google.com/presentation/d/138TkIVXTaRS3ciGYZ8A62q1QVaoA0ACcTmqVDjFv8-4/edit#slide=id.gced263eb5d_0_167)
+### Things to Remember
+
+- **No browser fully supports ES6**, to check browser support [Can I Use](https://caniuse.com/) is a great resource. 
```

Comment:
💙

Reply:
Thank you for your suggestions Alina and so sorry for the typos :(